content/en/docs/concepts/architecture/cri.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Container Runtime Interface (CRI)
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+The Kubernetes Container Runtime Interface (CRI) defines the main
```

Comment:
Better: add this or some similar text to the glossary, and then reuse that here with a `glossary_definition` shortcode.

Or use the same text twice (not shortcode).

nit: is the CRI part of Kubernetes/

Reply:
Note to myself: I'll do this next week, I'm OOO till Monday